src/components/reusable/CustomModal.js

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import Modal from 'react-modal';
+
+Modal.setAppElement("#root")
+
+export default function CustomModal({ Component, isOpen, onRequestClose, ...rest }) {
+    return (
+        <React.Fragment>
```

Comment:
I think Fragments here are unnecessary. There's also a new shorthand syntax for fragments - `<>` and `</>`